build-artifacts.py

```diff
@@ -22,17 +23,21 @@
     user_id=cfg.jenkins.user_id,
     api_token=cfg.jenkins.api_token)
 
-log.info("[ 1/5 ] Getting build info for change %s", change)
+output.info("[ 1/5 ] Getting build info for change %s", change)
 info = ga.build_info(change)
 
-log.info("[ 2/5 ] Starting build-artifacts job for %s", info)
+output.info("[ 2/5 ] Starting build-artifacts job for %s", info)
 queue_url = ja.run(
     url=info["url"], ref=info["ref"], stage="build-artifacts")
 
-log.info("[ 3/5 ] Waiting for queue item %s", queue_url)
+output.info("[ 3/5 ] Waiting for queue item %s", queue_url)
 job_url = ja.wait_for_queue(queue_url)
 
-log.info("[ 4/5 ] Waiting for job %s", job_url)
+output.info("[ 4/5 ] Waiting for job %s", job_url)
 result = ja.wait_for_job(job_url)
 
-log.info("[ 5/5 ] Job completed with %s", result)
+if result != "SUCCESS":
+    output.failure("Failed build artifcats %s", result)
```

Comment:
Lets keep this consistent with the message we show in system_tests.py

Reply:
Done